src/integrationTest/java/uk/gov/hmcts/reform/cwrdapi/StaffRefIntegrationTest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.cwrdapi;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.hmcts.reform.cwrdapi.util.AuthorizationEnabledIntegrationTest;
+
+class StaffRefIntegrationTest extends AuthorizationEnabledIntegrationTest {
+
+    @BeforeEach
+    public void setUpClient() {
+        super.setUpClient();
+    }
+
+    @Test
+    void returns_200_when_users_retrieved_successfully() throws JsonProcessingException {
+        //TODO:
```

Comment:
1. can you complete the test case and check for total records count from the response header
2. add another test case for non matching records with search string and check for total records count from the response header
3. add another test case for non staff admin user
4. add another test case with only with page number and check for total records count from the response header
5. add another test case with only size and check for total records count from the response header
6. add another test case for search string less than 3 chars
7. add another test case for null, empty and all possible special chars search string
8. add another test case with invalid page
9. add another test case with invalid size